manifests/v/Vencord/Vencord/Canary/1.2.10/Vencord.Vencord.Canary.locale.en-US.yaml

```diff
@@ -0,0 +1,18 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Vendicated.Vencord.Canary

```

Comment:
Hi @solomoncyj,

Should these be `Vendicated.Vencord.Canary` or `Vencord.Vencord.Canary`? This is the only location with `Vendicated`.

Reply:
Project member here: `Vendicated.Vencord.*` is the correct identifier as we only use `Vencord.*` for organisation projects, whilst Vencord itself (this project) is under Ven's personal user account. The other manifests that have been submitted are incorrect by our standards.